templates/zerver/emails/email.css

```diff
@@ -147,14 +147,10 @@ p.digest_paragraph,
 .sponsorship_request_link:hover,
 .support_request_link:hover,
 .email-preferences a:hover,
-.missed_message.no_content a:hover {
+.missed_message.no_content-explanation a:hover {
```

Comment:
Fixed this in 95c81ba0298ef5d368cb62a05b5a5dc3fd42ce9d.

(Your version still had a typo as best I can tell). Also, this kind of bug fix should definitely be in a separate commit from deleting stale CSS.

Reply:
Oops, I was so focused on adding the `explanation` that I overlooked checking the entire selector for accuracy. Thank you for catching that mistake.

Since the issue has been fixed, these changes are not included in this PR, which remains with only one commit.